cves/kernel/CVE-2020-15393.yml

```diff
@@ -391,37 +391,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
     note:
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
-    note:
+    applies: false
+    note: I can't research this one because Googling it brings up news about actual serial killers, and we haven't covered it in class.
```

Comment:
lol, similar experience

Reply:
Yeah, I tried to get more specific but it kept bringing up real-life murder cases.